teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person_class'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] This is a close one as well, but since the `Teacher` class inherits from the `Person` class it's important to ensure that all the necessary parameters are included in the constructor. I noticed that the `parent_permission` parameter is missing. so kindly add the parameter at line `4` with the default value set to `true`.